packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -552,7 +553,7 @@ export const components = ({
           borderRadius: odysseyTokens.BorderRadiusMain,
           borderColor: "transparent",
           fontSize: odysseyTokens.TypographySizeBody,
-          fontWeight: odysseyTokens.TypographyWeightBodyBold,
+          fontWeight: 500,
```

Comment:
confirm this

Reply:
We need to include this manually, right now we're not pulling in 500

Reply:
Add a ticket to add a token for 500 and pull in the correct file

Reply:
Turns out we already had one: `TypographyWeightHeading` is 500. And since the consumer pulls in the font (or, in Storybook's case, we pull it in from Google Fonts) we don't need to pull this in manually. We already grab the 500 face from Google Fonts.